### ISSUE TITLE
fx-compat: Wooshy not working at all in 102 build

### ISSUE THREAD

**Issue by dstillman** [zotero/zotero#2735]
There seems to be some more general breakage here with our 102 build — Wooshy isn't working on anything, even the Browser Console, which works in Firefox 102.

From #2733

**Comment by dstillman** [comment]
Oh, Wooshy must only activate on known bundle identifiers. They "added support for Zotero" in a recent version, and it doesn't work in the beta either.

I'll contact them.

**Comment by dstillman** [comment]
Works fine with `org.zotero.zotero`, so not our issue
